src/extensions/spdlog/SpdlogToActs.h

```diff
@@ -4,38 +4,118 @@
 
 #pragma once
 
+#include <boost/assign.hpp>
+#include <boost/bimap.hpp>
+
 #include <spdlog/spdlog.h>
 #include <spdlog/fmt/fmt.h>
 
 #include <Acts/Utilities/Logger.hpp>
 #include <JANA/JException.h>
 
 namespace eicrecon {
-    inline Acts::Logging::Level SpdlogToActsLevel(spdlog::level::level_enum input) {
-
-        // Convert the source string to lower case
-        switch (input) {
-            case spdlog::level::trace:
-                return Acts::Logging::VERBOSE;
-            case spdlog::level::debug:
-                return Acts::Logging::DEBUG;
-            case spdlog::level::info:
-                return Acts::Logging::INFO;
-            case spdlog::level::warn:
-                return Acts::Logging::WARNING;
-            case spdlog::level::err:
-                return Acts::Logging::ERROR;
-            case spdlog::level::critical:
-                return Acts::Logging::FATAL;
-            case spdlog::level::off:
-                return Acts::Logging::FATAL;
-            case spdlog::level::n_levels:
-                [[fallthrough]];
-            default:
-                break;
-        }
-
-        auto err_msg = fmt::format("SpdlogToActsLevel don't know this log level: '{}'", input);
-        throw JException(err_msg);
+
+using namespace Acts::Logging;
+
+using SpdlogToActsLevel_t = boost::bimap<spdlog::level::level_enum, Acts::Logging::Level>;
+static SpdlogToActsLevel_t kSpdlogToActsLevel = boost::assign::list_of<SpdlogToActsLevel_t::relation>
+  (spdlog::level::trace, Acts::Logging::VERBOSE)
+  (spdlog::level::debug, Acts::Logging::DEBUG)
+  (spdlog::level::info, Acts::Logging::INFO)
+  (spdlog::level::warn, Acts::Logging::WARNING)
+  (spdlog::level::err, Acts::Logging::ERROR)
+  (spdlog::level::critical, Acts::Logging::FATAL)
+  (spdlog::level::off, Acts::Logging::FATAL);
```

Comment:
Will this even work? https://www.boost.org/doc/libs/1_73_0/libs/bimap/doc/html/boost_bimap/the_tutorial/differences_with_standard_maps.html

Reply:
Use `std::option<Acts::Logging::Level>` or give up on boost?

Reply:
Ah, didn't even notice the double right value. We should be able to have a unique mapping.

Reply:
Surely doesn't work right now. All acts logs are gone entirely.

Reply:
Maybe we just don't need the bidirectional map anymore. We used to need it to translate the spdlog level to a corresponding acts level. Now we can probably just use a unidirectional map from acts to spdlog and be done with it.

Reply:
Bidirectional map retained for now, but duplication in right side removed.